containers/ncbi_datasets_v16.10.0.def

```diff
@@ -0,0 +1,55 @@
+Bootstrap: docker
+From: ubuntu:24.04
+
+%environment
+    export SINGULARITY_SHELL=/bin/bash
+    export DEBIAN_FRONTEND=noninteractive
+    export LC_ALL=C
+
+%labels
+    Author lcampbell@ebi.ac.uk
```

Comment:
I'm more than happy with this, just be sure you have a good spam/junk filter!

Reply:
I've added the ensembl-metazoa email instead. Is that alright or would you prefer complete anonymity ?